bootstrap/test_bootstrap.py

```diff
@@ -31,6 +31,15 @@
             }
         },
         "privileged": true
+    },
+    "ttyd": {
+        "tag": "master",
+        "image": "bluerobotics/companion-ttyd",
+        "enabled": true,
+        "webui": false,
+        "network": "host",
+        "binds": {},
```

Comment:
we need to bind the port and use privileged for ttyd.

Reply:
updated to set privileged=true.
we don't need to bind ports because we need network=host to connect to the pi's ssh